trinity/main.py

```diff
@@ -144,20 +157,18 @@ def main() -> None:
         try:
             initialize_data_dir(chain_config)
         except AmbigiousFileSystem:
-            exit_because_ambigious_filesystem(logger)
+            parser.error(TRINITY_AMBIGIOUS_FILESYSTEM_INFO)
         except MissingPath as e:
-            msg = (
+            parser.error((
```

Comment:
I think this would be nicer with a multiline f-string as it:

- doesn't need the `\n`
- just needs one pair of triple quotes at start / end
- can interpolate the `e.path` and get rid of the `format(e.path)` 

```python
parser.error(
    f"""
    It appears that {e.path} does not exist.
    Trinity does not attempt to create directories outside of its root path
    Either manually create the path or ensure you are using a data directory
    inside the XDG_TRINITY_ROOT path"""
)
```